platforms/corda-ent/charts/generate-pki/values.yaml

```diff
@@ -111,3 +111,15 @@ pki:
     networkmap:
       # subject: "CN=Test Network Map Service Certificate, OU=HQ, O=HoldCo LLC, L=New York, C=US"
       subject:
+    # Bridge certificate specifications
+    bridge:
```

Comment:
Will this mean with every new node in Corda, the generate-pki chart will be needed to be updated? Or will all the nodes in Corda use the same bridge and float component?

Reply:
This code is the baseline to check the workings of Bridge + Float. As discussed, the code is pushed to `develop` to make sure every developer has working code. Issue #898 will work on creating a dynamic PKI generator chart for all non-CENM related resources (Node, Float, Bridge). The certificates will be generated for each organization & node in the associated `values.yaml` file.  This code will be removed from the 'original' `generate-pki` upon completion of #898.